lib/payloadAction.ts

```diff
@@ -77,7 +78,7 @@ export const handlePayloadActions = async (
 				} else if (actionName === ActionIds.CLOSE_CHAT) {
 					const livechatRoom = (await read.getRoomReader().getById(rid)) as ILivechatRoom;
 					if (livechatRoom && livechatRoom.isOpen) {
-						await closeChat(modify, read, rid);
+						await closeChat(modify, read, rid, app, CloseChatDescription.CLOSED_BY_VISITOR);
```

Comment:
This is also confusing, it's closed when we receive close chat action from DF Agent. So it's not closed by visitor